packages/core/saved-objects/core-saved-objects-api-server-internal/tsconfig.json

```diff
@@ -4,8 +4,6 @@
     "declaration": true,
     "emitDeclarationOnly": true,
     "outDir": "target_types",
-    "rootDir": ".",
-    "stripInternal": false,
```

Comment:
There were some local linting issues e.g. :
> File '/.../packages/kbn-es-query/index.ts' is not under 'rootDir' '/.../packages/core/saved-objects/core-saved-objects-api-server-internal'. 'rootDir' is expected to contain all source files.

This change was suggested by @spenceralger to resolve the issue. Because the issue was local only, perhaps I should not have committed it.

Reply:
@jeramysoucy my instinct is that it's ok but let's get an "official" response to confirm.